### ISSUE TITLE
Blog

### ISSUE THREAD

**Issue by maurolepore** [ropenscilabs/pkginspector#13]
I copy Stefanie's email here to facilitate discussion.

---------- Forwarded message ----------
From: Stefanie Butland <stefanie@ropensci.org>
Date: Thu, Jun 7, 2018 at 6:24 PM
Subject: guest rOpenSci blog post on pkginspector /pkgtests?
To: Omayma Said <omayma.saeed@gmail.com>, "Albers, Sam ENV:EX" <Sam.Albers@gov.bc.ca>, Leonardo Collado Torres <leo.collado@libd.org>, Mauro Lepore <maurolepore@gmail.com>, Joyce Robbins <jtr13@columbia.edu>, Noam Ross <noam.ross@gmail.com>

Hi all,

What do you think of writing a guest post about your unconf18 pkginspector/pkgtests project for the rOpenSci blog? Please read on, decide among yourselves and then give me a yes or no with an approximate date for draft submission so we can review the post and give feedback before publishing.

If you're interested, when considering a theme for the post, it's good to lay out the motivation and approach and don't try to sound like you've managed to solve the problem with two days work :-)

Here are some project posts from unconf17: https://ropensci.org/tags/unconf17/ as examples. It would be ideal if everyone who contributed to the group is an author, though for some this could involve only reading and approving the draft post. Please make sure I didn't miss anyone on this email.

Here's the technical info: https://github.com/ropensci/roweb2#contributing-a-blog-post. 
Please include these tags, among others you might choose:
- r
- package
- community
- unconf
- unconf18

Happy to answer any questions.

Cheers!

Stef Butland

**Comment by maurolepore** [comment]
I'm up for this. This weekend I'll read the examples and instructions that Stefanie shared, and I'll read through all the great work you have been doing. Then I may have some thoughts to share or questions to ask.

**Comment by OmaymaS** [comment]
I definitely would like to contribute or right a draft :)
I think we address several points:

- **The motivation**
The questions reviewers/developers always have in mind while reviewing packages. How we try to facilitate this and what problem we are trying to solve.

- **The thought process**
How we started, what resources we started to look at, the research phase, how we wanted to build on another package, etc.

- **The oucome**
We can highlight that, we came up with sth integrated with pkgreviewr but then we decided to go for an independent package which is under development now

- **Next steps**
Our vision for the package and a call to action to have inputs from anyone about the hurdles, expectations, standards in package reviews
